gradle/versions.gradle

```diff
@@ -16,7 +16,7 @@ ext.versions = [
 	'xtend_lib': '2.28.0',
 	'guava': '[30.1,31)',
 	'guava_orbit': '30.1.0.v20221112-0806',
-	'gson': '[2.10.1,2.11)',
+	'gson': '[2.9.1,2.11)',
 	'gson_orbit': '2.10.1.v20230109-0753',

```

Comment:
afaik not anymore. just as a reference for the tycho stuff under releng.

Reply:
i also expect the other places to be adjusted e.g. feature.xml

Reply:
> afaik not anymore. just as a reference for the tycho stuff under releng.

OK, removed it. Lets see if the build succeeds.



> i also expect the other places to be adjusted e.g. feature.xml

Thanks for the hint. Adjusted the feature too and changed the version rule to compatible to not bind it to gson 2.9.
But I wonder why there is a dedicated dependency for gson in the feature at all?
Since the feature contains org.eclipse.lsp4j already it implicitly requires gson already anyway.
And p2 will consider the version specified in org.eclipse.lsp4j when installing the feature. Therefore this looks like a duplicated information to me. Probably the same for xtend.lib?

In general Feature-dependencies are actually only useful/necessary if none of the included Plug-ins requires it.